app/brave_settings_strings.grdp

```diff
@@ -1273,6 +1273,12 @@
   <message name="IDS_SETTINGS_IPFS_AUTO_FALLBACK_TO_GATEWAY_DESC" desc="The description for the switch of automatically fallback to IPFS public gateway">
     Automatically fall back to public IPFS gateway if your Brave local IPFS node cannot be reached.
   </message>
+  <message name="IDS_SETTINGS_IPFS_ALWAYS_START_MODE_LABEL" desc="The text label for IPFS always start mode switch">
+    IPFS always start mode
+  </message>
+  <message name="IDS_SETTINGS_IPFS_ALWAYS_START_MODE_DESC" desc="The description for the switch of IPFS always start mode switch">
+    Starts the IPFS node together with Brave
+  </message>
```

Comment:
Not sure if we need both the label and the description here. Maybe just have the label that says `Always start an IPFS node on the browser startup`?

Reply:
cc: @rmcfadden3

Reply:
Suggest `Always start an IPFS node on browser startup?`

Reply:
fixed